### PR TITLE
Code Review of Q5

### PR DESCRIPTION
**Pull request by Harshwardhanwtagline**
Screenshot 1944-11-25 at 3 42 42 AM
OutPut of Code

### CODE REVIEW

File: Q5.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from itertools import permutations
+
+
+def list_sum(num_list, user_input):
+    #I have use itertools-library to get permutations of different element 
+    permutations_list = list(permutations(num_list, 2))
+    #i have separated the combination of list , who's sum is equal to user_input and convert into list
+    sum_list = [list(_) for _ in set(permutations_list) if sum(_) == user_input]
```
Comment (himanshu-suthar-tagline) [inline Q5.py]: Please change `_` everywhere you used. You should use this only when there is no use of `_` in the code

File: Q5.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    #after then i have filter the dulipcate on base of condition.
+    result = list(filter(lambda x: x[0]< x[1] , sum_list))
```
Comment (himanshu-suthar-tagline) [inline Q5.py]: Remove extra space before the comma.

File: Q5.py (hunk 1 above)
Comment (Harshwardhanwtagline) [reply Q5.py]: Done Sir i have change it

File: Q5.py (hunk 2 above)
Comment (Harshwardhanwtagline) [reply Q5.py]: Done Sir, I have remove space from it.

**Pull request merged**